src/components/GameItem.js

```diff
@@ -20,11 +49,14 @@ class GameItem extends Component {
     };
 
     // Update this to select the correct icon for each item
-    const icon = ItemIcons.rock;
+    const icon = ItemIcons[this.props.type];
+
 
     return (
-      <div className="game-item" style={itemStyle}>
-        <img src={icon} alt="Item" className="icon-item"></img>
+      
+
+      <div className={this.state.spottedState} style={itemStyle}>
+        <img  src={icon} alt="Item" className="icon-item" onClick={this.onItemClicked}></img>
```

Comment:
To get a better chance of the click registering, I would put the `onClick` in the div tag rather than the img tag.